package.json

```diff
@@ -12,6 +12,9 @@
     "node": ">= 12.0.0",
     "npm": ">=6.0.0"
   },
+  "publishConfig": {
```

Comment:
[This can also be set in the `.npmrc` file](https://semantic-release.gitbook.io/semantic-release/support/faq#how-can-i-set-the-access-level-of-the-published-npm-package)

Reply:
I think I like it here since it specifically deals with publishing